packages/replay/test/unit/util/createReplayEnvelope.test.ts

```diff
@@ -0,0 +1,61 @@
+import { Event } from '@sentry/types';
+import { createReplayEnvelope } from '../../../src/util/createReplayEnvelope';
+
+describe('createReplayEnvelope', () => {
+  it('creates an envelope for a given Replay event', () => {
+    const replayId = '1234';
+    const replayEvent = {
+      type: 'replay_event',
+      timestamp: 1670837008.634,
+      error_ids: ['errorId'],
+      trace_ids: ['traceId'],
+      urls: ['https://example.com'],
+      replay_id: 'eventId',
+      segment_id: 3,
+      platform: 'javascript',
+      event_id: 'eventId',
+      environment: 'production',
+      sdk: {
+        integrations: ['BrowserTracing', 'Replay'],
+        name: 'sentry.javascript.browser',
+        version: '7.25.0',
```

Comment:
Will this need to be updated after every version bump?

Reply:
No, that's just random test data. Could be any version string

Reply:
Ahh sorry misread